packages/lib/team/service.ts

```diff
@@ -208,3 +242,31 @@ export const deleteTeam = async (teamId: string): Promise<TTeam> => {
     throw error;
   }
 };
+
+export const getTeamsWithPaidPlan = async (): Promise<TTeam[]> => {
+  try {
+    const teams = await prisma.team.findMany({
+      where: {
+        OR: [
+          {
+            billing: {
+              path: ["features", "appSurvey", "status"],
+              not: "inactive",
+            },
+          },
+          {
+            billing: {
+              path: ["features", "userTargeting", "status"],
+              not: "inactive",
+            },
+          },
+        ],
+      },
+      select: { ...select, products: { select: { environments: true } } }, // include environments
```

Comment:
Now I understand how you use it in the cron but the select conflicts with the return type TTeam. Let's try to loop through all products in the cron. Maybe if we do this async it can work.